app/components/requests_feed/requests_feed.html.erb

```diff
@@ -19,7 +19,7 @@
         <li>
           <a
             class="Link"
-            href="<%= request_path(reply.request, anchor: "contributor-#{contributor.id}") %>"
+            href="<%= request_path(reply.request, anchor: "message-#{reply.id}") %>"
```

Comment:
This was updated for consistency and because this PR would change the way it operated before this bug was introduced in [Optimize requests show page #1791](https://github.com/tactilenews/100eyes/pull/1791) .

There are currently no styles to highlight the ChatMessageGroup with like we do for MessageGroup and the RequestRow.

With these changes, there would be a blue border around the entire ChatMessageGroup instead of just the latest message in that group.